src/Pages/FurnitureDetail.jsx

```diff
@@ -0,0 +1,345 @@
+                        &#8377;{currentFurniture.price}/mo
+                      </Text>
+                      <Text fontSize="10px">Monthly Rent </Text>
+                    </Flex>
+                    <Flex direction="column">
+                      <Text fontSize="20px" fontWeight="500">
+                        {/* current product price + 150 */}
+                        &#8377; {Number(currentFurniture.price) + 150}/mo
+                      </Text>
+                      <Text fontSize="10px">Refundable Deposit </Text>
+                    </Flex>
+                  </Flex>
+
+                  <Flex fontSize="10px" padding="10px">
+                    {" "}
+                    &#10003; 7 Days free trial &#10003; Free relocation &#10003;
+                    Free upgrades
+                  </Flex>
+
+                  <Button color="teal" varient="outline" width="100%">
+                    Compare All Tenures
+                  </Button>
+
+                  <Flex direction="row" gap="10px" padding="10px">
+                    <Image src="https://www.rentomojo.com/public/images/fast-delivery/fast-delivery.svg" />
+                    <Text fontSize="10px">Delivery by {date}</Text>
+                  </Flex>
+                  <Button bg="red.500" color="white" width={"100%"} onClick={handleCart}>
+                    {/* ADD to Cart button */}
+                    Book your Plan
+                  </Button>
+
+                  <Box>
+                    <Flex justifyContent="space-between" p="10px">
+                      <Text>Special Offers</Text>
+                      <Text color="blue.300">+1 More Offers</Text>
+                    </Flex>
+                    <Flex
+                      border="2px solid #e9e9e9"
+                      justifyContent="space-evenly"
+                      alignItems="center"
+                      p="10px"
+                      gap="10px"
+                    >
+                      <Image
+                        src="https://www.rentomojo.com/public/images/product/special-offer.svg"
+                        htmlWidth="30px"
+                      />
+                      <Flex fontSize="10px" direction="column">
+                        <Text>
+                          Use Code <strong>CITY15</strong>
+                        </Text>
+                        <Text>Flat 15% off on third Month Invoice </Text>
+                        <Text color="blue.400">Read More</Text>
+                        <Text color="red.400">Applicable to New users.</Text>
+                      </Flex>
+                      <Button variant="outline" color="blue.600">
+                        Copy
+                      </Button>
+                    </Flex>
+                  </Box>
+                </Box>
+              </Flex>
+            </Box>
+          </Flex>
+        </Box>
+      </Box>
+    </div>
+  );
+};
+
+export default FurnitureDetail;
```

Comment:
You are repeating the fitness detail code here. That can be easily resuble. You cannot have a detail page for every category. Work on this